lib/linebuffer.js

```diff
@@ -133,7 +133,9 @@ const flush = () => {
                         url: config.LOGDNA_LOGURL + config.LOGDNA_LOGENDPOINT
                         , method: 'POST'
                         , body: out
-                        , headers: config.DEFAULT_REQ_HEADERS_GZIP
+                        , headers: Object.assign({}, config.DEFAULT_REQ_HEADERS_GZIP, {
+                            'user-agent': config.UA
```

Comment:
Am I missing something? I don't see `config.UA` defined anywhere. Let me know if I'm wrong.

Reply:
@dchai76 https://github.com/logdna/logdna-agent/blob/v1.6.2/index.js#L36

Reply:
OK wow, that's confusing. I strongly suggest we move the `package` include into the `config`, define `UA` as a field there (and let's spell it out - `USER_AGENT`), and include that field specifically in `DEFAULT_REQ_HEADERS_GZIP` and `DEFAULT_REQ_HEADERS` in the config instead of augmenting here. The point of a centralized config file is to centralize config, not have it distributed in different places, so let's do that.

Reply:
That's good idea! I'll keep it as `UserAgent` - consistent across all other integrations

Reply:
Updated! Done!